Source/ARTRealtime.m

```diff
@@ -643,6 +644,7 @@ - (ARTEventListener *)transitionSideEffects:(ARTConnectionStateChange *)stateCha
             break;
         }
         case ARTRealtimeSuspended: {
+            _retryCount = 0;
```

Comment:
When I was looking at the feature spec, it wasn't clear to me where it says that the retry counter should be reset in certain circumstances. ([RTB1a](https://sdk.ably.com/builds/ably/specification/main/features/#RTB1a) talks about the "nth retry" but the nth retry since what?) 

I was wondering what made you choose to reset it in the places that you have (both in the case of `ARTRealtime` and `ARTRealtimeChannel`? I think that the choice that you've made (reset it once a `CONNECTING` / `ATTACHING` <-> `SUSPENDED` cycle ends) makes sense intuitively, but was there something in the spec that led you to it?

Reply:
I took it from [this discussion](https://github.com/ably/ably-js/pull/1008#discussion_r925947944)

Reply:
I think it would be good for this to be more properly specified instead of us relying on the implementation of another SDK.

Would you be able to open an issue in the specification repo for this, please? I don't think it needs to block us merging this PR, though.

Reply:
I've created an issue https://github.com/ably/specification/issues/127